tests/test_all_rules.py

```diff
@@ -59,7 +59,10 @@ def test_file_names(self):
     def test_all_rule_queries_optimized(self):
         """Ensure that every rule query is in optimized form."""
         for rule in self.production_rules:
-            if rule.contents.data.get("language") == "kuery":
+            if (
+                rule.contents.data.get("language") == "kuery" and "powershell.file.script_block_text"
```

Comment:
```suggestion
                rule.contents.data.get("language") == "kuery" and "powershell.file.script_block_text"
```

Can we add this to `definitions.py` as `QUERY_FIELD_OP_EXCEPTIONS = []`. Then import this and use here? More of a nit but just keeps code consistent.

Reply:
@eric-forte-elastic

Reply:
Added to definitions and imported :+1: